flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/batch/sql/FunctionITCase.java

```diff
@@ -199,4 +265,48 @@ private void testUserDefinedFunctionByUsingJar(String createFunctionDDL, String
         // delete the function
         tEnv().executeSql(dropFunctionDDL);
     }
+
+    private void testUserDefinedFunctionByUsingJar(FunctionCreator creator, String dropFunctionDDL)
```

Comment:
I think we can delete `testUserDefinedFunctionByUsingJar(String createFunctionDDL, String dropFunctionDDL)` method, all related test use this method.